tests/sites/mapping/devices/GAT-123/expected/exceptions.txt

```diff
@@ -1,3 +1,6 @@
+Config
+java.lang.NullPointerException: Cannot read field "min_loglevel" because "this.metadata.system" is null
```

Comment:
This doesn't seem right...

Reply:
I believe this is unchanged behaviour except the exception is now surfaced. It's expected because the metadata file is invalid, so metadata becomes `null`